tests/test_requests.py

```diff
@@ -166,8 +174,6 @@ def test_create_participant_team_for_object_does_not_exist(self):
         runner = CliRunner()
         result = runner.invoke(teams, ['create'], input="TeamTest\ny\n")
         response = result.output.rstrip()
-        user_prompt_text = ("Enter team name: : TeamTest\n"
```

Comment:
@isht3 Why you have removed this line?

Reply:
It's not changed. It just went down a few lines. I don't understand why git's showing it different.